astropy/cosmology/core.py

```diff
@@ -777,6 +791,46 @@ def nu_relative_density(self, z):
 
         return prefac * self._neff_per_nu * rel_mass
 
+    def _get_nufunc_scalar(self):
+        """ Return the version of the nu relative density function
+        to use in integrals."""
+
+        if self._massivenu:
+            return self._massivenu_relative_density_scalar
+        else:
+            return lambda x : 0.22710731766 * self._Neff
```

Comment:
Since this is the same constant used [here](https://github.com/astropy/astropy/pull/3814/files#diff-aa1aaec38c2ad7de67b2b1c1a889a6f8R769) maybe just make it a module (or class) level constant?